packages/graphql/schemas/schema.graphql

```diff
@@ -598,9 +598,6 @@ type ProjectPreferences {
 type Query {
   baseError: BaseError
 
-  """An error related to finding a browser"""
-  browserErrorMessage: String
```

Comment:
I believe the reason this was called out as a `browserErrorWarning` was so that we knew where to display it. For instance if they passed `cypress open --browser invalid-browser` we wouldn't show the warning until they go to the display browser screen.

A lot of the management of state is going to change in #18906 though, so I think we can keep it as-is for now.

Reply:
Yeah, it's not strictly necessary to make the warnings a list for the sake of the browser warning, but apparently there will be some need for multiple warnings in the future, so I figured this would fit in better if I went ahead and implemented the browser warning as part of a list. I added the `setupStep` property so warnings could specify which page of the wizard they should appear on.